config/manager/manager.yaml

```diff
@@ -33,6 +33,7 @@ spec:
           - --leader-elect
           - "--diagnostics-address=${CAAPH_DIAGNOSTICS_ADDRESS:=:8443}"
           - "--insecure-diagnostics=${CAAPH_INSECURE_DIAGNOSTICS:=false}"
+          - "--sync-period=${CAAPH_SYNC_PERIOD:=10m}"
```

Comment:
Is this change related to the rest of the PR?

Reply:
Yes, I added this so in the test, we could change the sync period to 1m, so that when we have the `time.Sleep()` after an out of band upgrade, we can be sure a reconciliation loop was triggered.